apple/internal/codesigning_support.bzl

```diff
@@ -592,6 +602,7 @@ def _sign_binary_action(
         paths_to_sign = [path_to_sign],
         platform_prerequisites = platform_prerequisites,
         provisioning_profile = provisioning_profile,
+        codesignopts = [],
```

Comment:
does it also need to be threaded through to this action?

Reply:
this one doesn't come from a codepath that can have one atm, so i just ignored it